packages/react/src/KvibProvider.tsx

```diff
@@ -1,25 +1,15 @@
-import { ChakraProvider, ChakraProviderProps, extendTheme } from "@chakra-ui/react";
+import "@fontsource/mulish/400.css";
+import "@fontsource/mulish/600.css";
+import "@fontsource/mulish/700.css";
+
```

Comment:
Kult, nytt for meg det her. Søkte det opp og de mener det er en ytelsesforbedring, så det er kjekt